recipes/sqlite_orm/all/conanfile.py

```diff
@@ -40,7 +40,7 @@ def layout(self):
         basic_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("sqlite3/3.45.0", transitive_headers=True, transitive_libs=True)
+        self.requires("sqlite3/[^3.45]", transitive_headers=True, transitive_libs=True)
```

Comment:
```suggestion
        self.requires("sqlite3/[>=3.45.0 <4]", transitive_headers=True, transitive_libs=True)
```

(so that it matches the range as we have it elsewhere)